package.json

```diff
@@ -37,26 +37,26 @@
   "author": "@djh",
   "license": "Apache-2.0",
   "dependencies": {
-    "@unocss/core": "^0.50.6",
-    "@unocss/preset-mini": "^0.50.6"
+    "@unocss/core": "^0.55.1",
+    "@unocss/preset-mini": "^0.52.7"
```

Comment:
This dependency has to be at 0.52.7 for now. 
In [0.53.0](https://github.com/unocss/unocss/pull/2714) pseudo variants generation order changes.
That causes changes to css order for classes that control the same CSS properties,
which results in bugs in our components (e.g. Button disabled gets a non-disabled text color).
Before we update this package, we should change the way we assign utility classes to components
when they control the same CSS properties.

Reply:
Nice, here's the ticket to do that btw: https://nmp-jira.atlassian.net/browse/WARP-231